firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -1147,7 +1147,19 @@ class BrowserCoordinator: BaseCoordinator,
 
     func showStoriesWebView(url: URL?) {
         guard let url else { return }
-        let tab = tabManager.addTab(URLRequest(url: url), afterTab: nil, zombie: false, isPrivate: false)
+
+        // Creates an unmanaged tab that is destroyed once the stories webview view controller is dismissed
+        // Used to prevent persisting tab in the tab tray during and across app sessions
+        let tab = Tab(profile: profile, isPrivate: false, windowUUID: windowUUID)
+        let tabConfigurationProvider = TabConfigurationProvider(prefs: profile.prefs)
+        let tabConfiguration = tabConfigurationProvider.configuration(isPrivate: tab.isPrivate).webViewConfiguration
+        tab.url = url
+        tab.navigationDelegate = browserViewController
+        tab.tabDelegate = browserViewController
+        tab.createWebview(configuration: tabConfiguration)
+        tab.loadRequest(URLRequest(url: url))
```

Comment:
@MattLichtenstein Hello, sorry I missed this PR, but this responsibility shouldn't be added into the coordinator. Coordinators should be simple objects that manages navigation. This feels like too much responsibility and knowledge for a coordinator, can we move this code into a manager of some sort/somewhere else and add unit tests for this? Thank you!